### PR TITLE
Update MAC_SETUP.md

### PR DESCRIPTION
**Pull request by outsidecoder**
Added a testing section for the SSH setup.

### What

Describe what you have changed and why.

### How to review

Describe the steps required to test the changes.

### Who can review

Describe who worked on the changes, so that other people can review.

### CODE REVIEW

**Review by DavidSubiros — DISMISSED** [review]
I validated that `ssh -T git@github.com` returns the message. Thanks for providing this SSH testing description!

File: guides/MAC_SETUP.md [hunk 1]
```diff
@@ -130,6 +130,16 @@ Then follow [instructions to add the SSH key in the clipboard to your github acc
 
 Return to the [Getting Started](https://github.com/ONSdigital/dp/blob/master/guides/GETTING_STARTED.md) guide for next steps.
 
+### Testing SSH Connection
+
+To check that you've setup your Git SSH on your terminal open your favoured terminal and enter:
```
Comment (dandwal) [inline guides/MAC_SETUP.md:135]: - setup -> set up
(noun -> verb)
- comma 

```suggestion
To check that you've set up your Git SSH on your terminal, open your favoured terminal and enter:
```

File: guides/MAC_SETUP.md [hunk 2]
```diff
@@ -130,6 +130,16 @@ Then follow [instructions to add the SSH key in the clipboard to your github acc
...
+  * `ssh -T git@github.com`
+  
+You will then be asked to type *yes* if you're ready to connect.  You should then be then be prompted with:
```
Comment (dandwal) [inline guides/MAC_SETUP.md:138]: - Double `then be` we can remove one
- Removal of double spaces (think the rest of our document uses single space after full-stop so just a consistency thing although optional)
```suggestion
You will then be asked to type *yes* if you're ready to connect. You should then be prompted with:
```

File: guides/MAC_SETUP.md [hunk 3]
```diff
@@ -130,6 +130,16 @@ Then follow [instructions to add the SSH key in the clipboard to your github acc
...
+  * `Hi [YOUR GIT USERNAME]! You've successfully authenticated, but GitHub does not provide shell access.`
+
+You can then close your terminal.  If its unsuccessful check you've used the correct email address (email used for Github account) when creating and saving your key.
```
Comment (dandwal) [inline guides/MAC_SETUP.md:141]: - Removal of double spaces (think the rest of our document uses single space after full-stop so just a consistency thing although optional)
- Added apostrophe to `its`
```suggestion
You can then close your terminal. If it's unsuccessful check you've used the correct email address (email used for Github account) when creating and saving your key.
```

**Review by dandwal — COMMENTED** [review]
Very good and clear instructions, a few grammar points raised however not blocking the approval (but would be good if we removed the double `then be` and then happy to approve)

**Review by DavidSubiros — APPROVED** [review]

**Review by dandwal — APPROVED** [review]
LGTM

**Pull request merged**